### PR TITLE
New version: WolframResearch.WolframEngine version 13.0.0

### PR DESCRIPTION
**Pull request by cc-wr**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41697)

### CODE REVIEW

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- mathematica
+- wolfram
+- wolframengine
+- wolfram-engine
+- free
+- developers
+PackageUrl: https://www.wolfram.com/engine/
+License: Copyright (c) Wolfram Research, Inc. All rights reserved.
+LicenseUrl: https://www.wolfram.com/legal/terms/wolfram-engine.html
+PackageVersion: 13.0.0
```
Comment (OfficialEsco) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:17]:
```suggestion
PackageVersion: 13.0.282
```

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: WolframResearch.WolframEngine
+Publisher: Wolfram Research
```
Comment (OfficialEsco) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:2]:
```suggestion
Publisher: Wolfram Research, Inc.
```

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+PackageName: Wolfram Engine
```
Comment (OfficialEsco) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:3]:
```suggestion
PackageName: Wolfram Engine 13
```

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Author: Wolfram Research
+ShortDescription: Locally downloadable Wolfram Engine to add computational intelligence to pre-production software development.
+Moniker: WolframEngine
```
Comment (OfficialEsco) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:6]:
```suggestion
Moniker: wolframengine
```

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+- free
+- developers
+PackageUrl: https://www.wolfram.com/engine/
+License: Copyright (c) Wolfram Research, Inc. All rights reserved.
+LicenseUrl: https://www.wolfram.com/legal/terms/wolfram-engine.html
+PackageVersion: 13.0.0
+Installers:
+- Architecture: x64
+  InstallerUrl: https://files.wolframcdn.com/packages/winget/13.0.0.0/WolframEngine_13.0.0_WIN.msi
+  InstallerSha256: b1007bb2672a65441f9a8cfa412787b85e75d65d5e888544065c4b02c83cac1e
```
Comment (OfficialEsco) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:21]:
```suggestion
  InstallerSha256: b1007bb2672a65441f9a8cfa412787b85e75d65d5e888544065c4b02c83cac1e
  ProductCode: '{9CC972A1-5E5D-11EC-A34B-0CC47AC03162}'
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Remember to update version folder name

File: manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Tags:
+- mathematica
+- wolfram
+- wolframengine
+- wolfram-engine
+- free
+- developers
+PackageUrl: https://www.wolfram.com/engine/
+License: Copyright (c) Wolfram Research, Inc. All rights reserved.
+LicenseUrl: https://www.wolfram.com/legal/terms/wolfram-engine.html
```
Comment (Trenly) [inline manifests/w/WolframResearch/WolframEngine/13.0.0/WolframResearch.WolframEngine.yaml:16]:
```suggestion
License: Proprietary
LicenseUrl: https://www.wolfram.com/legal/terms/wolfram-engine.html
Copyright: Copyright (c) Wolfram Research, Inc. All rights reserved.
```

**Review by Trenly — CHANGES_REQUESTED** [review]
Along with Esco's suggestions, I would also suggest this be made as a multi-manifest to make it easier for maintaining in the future

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/41697#pullrequestreview-860095149

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**